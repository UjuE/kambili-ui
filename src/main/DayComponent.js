import React from 'react';

export default class DayComponent extends React.Component {
    render() {
        return (
            <div className="day">
                {this.props.value}
            </div>);
    }
}