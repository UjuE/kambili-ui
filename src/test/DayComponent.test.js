import React from 'react';
import { shallow } from 'enzyme';
import DayComponent from '../main/DayComponent';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });

describe("DayComponent", () => {
    it("should accept parameters", () => {
        const component = shallow(<DayComponent value="Sunday" />);
        expect(component.getElements()).toMatchSnapshot();
    });
});