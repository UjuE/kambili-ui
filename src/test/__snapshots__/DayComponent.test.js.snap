// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DayComponent should accept parameters 1`] = `
Array [
  <div
    className="day"
  >
    Sunday
  </div>,
]
`;
